Minor fixes (#35)
drtp-skills/www.drtp-skills.ac.uk: _pages/funds-prof-dev.md

The total value of this fund is £150k to be administered throughout the duration of the CHARTED project, with the maximum award per individual being £5k. The eligibility criteria and the review process are described below. 

**The fund is designed to support specific activities and directly associated costs only. Requests for equipment or human effort are not eligible.**

> Applicants should therefore gain approval from their institutions to cover the time spent on the activities for which they seek support.

Reviewer: ```suggestion
**The fund is designed to support specific professional development activities by reimbursing costs such as access to training courses or travel costs and registration fees for attending conferences or events that would support an individual's professional development. Requests for funding to cover staff time are not eligible.**
Applicants should therefore gain approval from their home institution to cover the time spent on the activities for which they seek support.
```

<li>Detailed breakdown for the expected costs and justification for the requested funds.</li>
</ul>

> By default this award will be administered following the reimbursement model, applicants will therefore be expected to keep good records of all expenses. Final reimbursement requests should be within 10% of the requested costs. If this model does not work for you, please indicate that clearly in your application. You must apply and be successful in your application before the activity you seek support for takes place.

Reviewer: ```suggestion
By default this award will be administered following the reimbursement model, applicants will therefore be expected to keep comprehensive records of all expenses. Final reimbursement requests should be within 10% of the requested costs. If this model does not work for you, please indicate that clearly in your application. You must apply and be successful in your application before incurring any costs in relation to the activity you are seeking support for.
```
